lib/dbvar/src/Data/DeltaMap.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE TypeFamilies #-}
+
+module Data.DeltaMap
+    ( DeltaMap(..)
+    ) where
+
+import Data.Delta
+    ( Delta (..) )
+import Data.Map.Strict
+    ( Map )
+import qualified Data.Map.Strict as Map
+import Prelude
+    ( Ord )
```

Comment:
A small request!
> Imports should be grouped into three groups, separated by a blank newline.
> -  Prelude import
> -  Explicit imports
> -  Qualified imports

I suspect that Stylish Haskell can't do this separation automatically. However, it will obey the groupings if they are there!

Source: https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md#stylish-haskell-is-used-to-format-grouped-imports--language-pragmas

```suggestion
import Prelude
    ( Ord )

import Data.Delta
    ( Delta (..) )
import Data.Map.Strict
    ( Map )
    
import qualified Data.Map.Strict as Map
```